.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "FV3"]
 	path = FV3
 	url = https://github.com/NOAA-EMC/fv3atm
-	branch = production/GFS.v16
+	branch = GFS.v16
```

Comment:
@HelinWei-NOAA The branch is: production/GFS.v16. Please update.

Reply:
Done